* When a ```FailureReason``` is returned to the ```LambdaHelper``` the ```FailureReason``` is serached for an HTTP error code (instead of always using 500).